mono/metadata/metadata.c

```diff
@@ -3962,9 +3963,10 @@ mono_method_get_header_summary (MonoMethod *method, MonoMethodHeaderSummary *sum
  *
  * Returns: a transient MonoMethodHeader allocated from the heap.
  */
-MonoMethodHeader *
+MONO_API MonoMethodHeader *
```

Comment:
If the declaration in the `.h` file is marked as `MONO_API` then there is no need to do it in the `.c` file too.